bl1/aarch64/bl1_arch_setup.c

```diff
@@ -38,7 +38,7 @@
 void bl1_arch_setup(void)
 {
 	/* Set the next EL to be AArch64 */
-	write_scr_el3(SCR_RES1_BITS | SCR_RW_BIT);
+	write_scr_el3(read_scr_el3() | SCR_RW_BIT);
```

Comment:
So you've moved initialization of the `SCR_RES1_BITS` into `el3_arch_init_common` for both BL1 and BL31, and changed this to use an SBOP (Should-Be-One-or-Preserved) policy for rewriting `SCR_EL3`. This looks correct but could do with some explanation in the commit message!